src/orchestrator.ts

```diff
@@ -208,7 +209,7 @@ export class Orchestrator {
         const taskFailed = this.findTaskFailed(state, taskScheduled);
         this.setProcessed([taskScheduled, taskCompleted, taskFailed]);
 
-        if (taskCompleted) {
+        if (taskCompleted !== undefined) {
```

Comment:
Why make the distinction between falsey values? Do we intend to treat `null` differently from `undefined`? I'm concerned that explicit checks like these could lead to subtle bugs if a `null` slips through somewhere and we don't correctly detect it.

Reply:
I think I was misinterpreting the advice given [here](https://basarat.gitbook.io/typescript/recap/null-undefined#final-thoughts), regarding not using `null`.

That being said, the same [section](https://basarat.gitbook.io/typescript/recap/null-undefined#checking-for-either) literally says to use `!= null`, as that properly checks for both, and says that you don't want to make a distinction between the two. 

I'm not sure if it is better condition to do general falsy checks like what existed before, or just do these `!= null` checks to only check for `null` or `undefined`. Knowing JS and the general appeal to brevity, I am presuming that the falsy check we used to have is preferred, so I will revert my change.